src/something_game.cpp

```diff
@@ -602,7 +614,7 @@ void Game::render_debug_overlay(SDL_Renderer *renderer, size_t fps)
 
     const float COLLISION_PROBE_SIZE = 10.0f;
     const auto collision_probe_rect = rect(
-        camera.to_screen(collision_probe - COLLISION_PROBE_SIZE),
+        camera.to_screen(mouse_position - COLLISION_PROBE_SIZE),
```

Comment:
Collision probe was created to visually test collision resolution. And now it always follow exact mouse position. I think it could stay as it is, not updated without mouse movement. Because when you debug collision, usually you don't need to move player.

Reply:
I rollback this change, as you suggested.